src/pages/ResearcherConsole.js

```diff
@@ -223,33 +224,35 @@ class ResearcherConsole extends Component {
             div({ isRendered: ResearcherConsole.partialDars !== 0, className: "row no-margin" }, [
               PageSubHeading({
                 id: "researcherConsoleSavedAccess",
-                color: "default",
+                color: "access",
                 iconSize: "none",
                 title: "Saved Data Access Requests"
               }),
-              div({ className: "jumbotron table-box" }, [
+              div({ style: Theme.lightTable }, [
                 div({ className: "row no-margin" }, [
-                  div({ className: "col-lg-2 col-lg-offset-1 col-md-2 col-md-offset-1 col-sm-2 col-sm-offset-1 col-xs-2 col-xs-offset-1 cell-header default-color" }, ["Temporary id"]),
-                  div({ className: "col-lg-5 col-md-5 col-sm-5 col-xs-5 cell-header default-color" }, ["Project title"]),
-                  div({ className: "col-lg-2 col-md-2 col-sm-2 col-xs-2 cell-header default-color" }, ["Date"]),
-                  div({ className: "col-lg-2 col-md-2 col-sm-2 col-xs-2 cell-header f-center default-color" }, ["Resume"]),
+                  div({ style: Theme.textTableHead, className: "col-xs-2 access-color" }, ["Temporary ID"]),
+                  div({ style: Theme.textTableHead, className: "col-xs-5 col-xs-offset-1 access-color" }, ["Project Title"]),
+                  div({ style: Theme.textTableHead, className: "col-xs-2 access-color" }, ["Date"]),
+                  div({ style: Theme.textTableHead, className: "col-xs-2 f-center access-color" }, ["Resume"]),
```

Comment:
<img width="1189" alt="Screen Shot 2021-01-15 at 2 20 08 PM" src="https://user-images.githubusercontent.com/43456581/104769222-f8dfc180-573c-11eb-9b29-6c0d4935cb14.png">
Looks like the heading for this table got misaligned here! The 1 col worth of space for the trash icon was provided by the `col-xx-offset-1`, but as an alternative you could add a new empty column placeholder, if you don't want to depend on the classes.